### PR TITLE
OD-508 [FIX] Fixed to saving size after moving columns

### PR DESCRIPTION
**Pull request by OleksandrLavrynenko**
@sofiiakvasnevska @inna-bieshulia

OD-508 https://weboo.atlassian.net/browse/OD-508

## Description
Fixed to saving size after moving columns. The column is now resized after being moved.

## Screenshots/screencasts
https://storyxpress.co/video/krqjvs9ehxp7ssjwe

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko @S-Volodymyr @skrupnyk

### CODE REVIEW

File: js/spreadsheet.js [hunk 1]
```diff
@@ -531,6 +531,26 @@ var spreadsheet = function(options) {
 
       colWidths = getColWidths();
     },
+    beforeColumnMove: function(items, index) {
+      colWidths = getColWidths();
+
+      var length = items.length;
+      var colsToMove = colWidths.splice(items[0], length);
```
Comment (squallstar) [inline js/spreadsheet.js:538]: any chance `items` can be empty here?

Comment (OleksandrLavrynenko) [reply js/spreadsheet.js:538]: > any chance `items` can be empty here?

the hook "beforeColumnMove" is activated when at least one item is selected

**Review by squallstar — APPROVED** [review]
@inna-bieshulia this is ready to be deployed on **staging** for QA.

- **Risk**: Medium
- **Affected**: Studio users using the data source manager
- **Verify**: Verify the issue is resolved

**Comment by OleksandrLavrynenko** [comment]
@tonytlwu could you please advice what we need to do with the function "cells"? It is an old code and Lint does not allow it. Should we rewrite it or add ignore?

**Comment by tonytlwu** [comment]
@OleksandrLavrynenko What do you mean by the Lint doesn't allow it? What's rule is it breaking and what's the error?

**Comment by upplabs-alex-levchenko** [comment]
@tonytlwu It's about Codacy Static Code Analysis, it says - "A function should not mix return statements with and without a result".

**Comment by tonytlwu** [comment]
@upplabs-alex-levchenko Thanks for the clarification. I'd ignore it by adding `// eslint-disable-next-line consistent-return`. I'm not sure which line to add it in (before the `function` definition or the `return` statement). I'll let you guys figure that out.

**Comment by upplabs-alex-levchenko** [comment]
Thanks, roger that.

**Comment by upplabs-alex-levchenko** [comment]
@tonytlwu we've tried different annotations and their positions but it does not affect Codacy for some reason. Please ignore this error through their UI cuz it's not available for us. Thanks. 
image

**Comment by tonytlwu** [comment]
@upplabs-alex-levchenko Let's keep this moving forward as it is then. We don't want to globally disable the rule because it's a good sense check for inconsistencies, e.g. some functions need to always return a Promise and it would be bad if one of the clauses returned `undefined`.

@inna-bieshulia You can merge this and continue with testing now.

**Pull request merged**